unittests/abi_tests.cpp

```diff
@@ -26,7 +26,7 @@ using namespace chain;
 
 BOOST_AUTO_TEST_SUITE(abi_tests)
 
-fc::microseconds max_serialization_time = fc::seconds(1); // some test machines are very slow
+fc::microseconds max_serialization_time = fc::seconds(10000); // some test machines are very slow
```

Comment:
10'000 seconds is almost 3 hours. do you really need here so long timeout?

Reply:
This change should be reverted.